index.js

```diff
@@ -11,8 +10,8 @@ const VERSION = 3; // serialized format version
 export default class Flatbush {
 
     static from(data) {
-        if (!(data instanceof ArrayBuffer)) {
-            throw new Error('Data must be an instance of ArrayBuffer.');
+        if (data.constructor.name !== 'ArrayBuffer' && data.constructor.name !== 'SharedArrayBuffer') {
```

Comment:
Sorry for being nitpicky — this will fail if the buffer is subclassed (`class MyBuffer extends ArrayBuffer {}`). I tried looking for a way to do this check cleanly and only found https://github.com/inspect-js/is-shared-array-buffer, which does too much complicated magic for our purposes, however, maybe in this case we could get away with some kind of simple duck typing? E.g. both buffer classes have `byteLength`, but unlike typed arrays & DataView they don't have `buffer`. So maybe something like this:

```suggestion
        if (!data || data.byteLength === undefined || data.buffer) {
```